generators/pattern-library/index.js

```diff
@@ -14,12 +14,17 @@ module.exports = init;
 
 function init( config ) {
 
+    // First, add errors array to config
+    config.errors = [];
+
     config = configure( config );
 
-    if ( Array.isArray( config ) ) {
+    // If we have errors in the array, reject and handoff those errors to the user
+    if ( config.errors.length ) {
+
+        return new Promise( ( resolve, reject ) => {
 
-        return new Promise( function ( resolve, reject ) {
-            return reject( config );
+            return reject( config.errors );
```

Comment:
When there is an error, the user won't be using anything else other than the errors, so there isn't a reason I can see why they would need the whole config.